aiomisc/service/raven.py

```diff
@@ -21,6 +21,51 @@ def send(self, url, data, headers):
         pass
 
 
+class QueuedKeepaliveAioHttpTransport(QueuedAioHttpTransport):
+    DNS_CACHE_TTL = 600
+    DNS_CACHE = True
+    TCP_CONNECTION_LIMIT = 32
+    TCP_CONNECTION_LIMIT_HOST = 8
+    WORKERS = 1
+    QUEUE_SISE = 1000
+
+    def __init__(self, *args, family=socket.AF_UNSPEC, loop=None,
+                 dns_cache: bool = DNS_CACHE,
+                 dns_cache_ttl: int = DNS_CACHE_TTL,
+                 connection_limit: int = TCP_CONNECTION_LIMIT,
+                 connection_limit_host: int = TCP_CONNECTION_LIMIT_HOST,
+                 workers: int = WORKERS, qsize: int = QUEUE_SISE, **kwargs):
+
+        super().__init__(
```

Comment:
In [`__init__.py` (`AioHttpTransportBase`)](https://github.com/getsentry/raven-aiohttp/blob/d576479db15dd594de75e1e65a7becccaad67a83/raven_aiohttp.py):
```
        if self.keepalive:
            self._client_session = self._client_session_factory()
```
So with keep_alive there will be an issue, move super() below attributes set.